packages/bot-web-ui/src/stores/root-store.ts

```diff
@@ -12,14 +12,15 @@ import SummaryCardStore from './summary-card-store';
 import ToolbarStore from './toolbar-store';
 import TransactionsStore from './transactions-store';
 import QuickStrategyStore from './quick-strategy-store';
-import MainContentStore from './main-content-store';
 import RoutePromptDialogStore from './route-prompt-dialog-store';
 import DataCollectionStore from './data-collection-store';
 import BlocklyStore from './blockly-store';
 import SelfExclusionStore from './self-exclusion-store';
 import ToolboxStore from './toolbox-store';
 import AppStore from './app-store';
+import DashboardStore from './dashboard-store';
 
+//Todo need to write types for the root store
```

Comment:
Just a suggestion: Use `// TODO ....` to map it in the vscode extension and better tracking.

Reply:
done ✅